lib/gold_miner/author.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module GoldMiner
+  Author = Data.define(:id, :name, :link) do
```

Comment:
I hate that `Author` doesn't quite fit the gold mining analogy. A gold nugget having an author doesn't really make sense, but I couldn't think of any other word.